src/app/bundles/bundle-page/bundle-page.component.ts

```diff
@@ -1,15 +1,30 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, Input } from '@angular/core';
+import { BundleDocument, State } from '../shared/bundle.interfaces';
+import { Store } from '@ngrx/store';
+import { Observable } from 'rxjs';
+import { StitchBundle } from './bundle-page.actions';
+import { ActivatedRoute, ParamMap } from '@angular/router';
 
 @Component({
   selector: 'app-bundle-page',
-  templateUrl: './bundle-page.component.html',
-  styleUrls: ['./bundle-page.component.scss']
+  templateUrl: './bundle-page.component.html'
 })
-export class BundlePageComponent implements OnInit {
+export class BundlePageComponent {
 
-  constructor() { }
+  @Input() documents$: Observable<BundleDocument[]>;
```

Comment:
is $ at the end of a variable some convention I'm not aware of?

Reply:
I don't have a strong opinion on this, but it's definitely useful because you can recognise it instantly as an observable, and also differentiate between the variables for the observable (documents$) and its value (documents)